tests/test_regex.py

```diff
@@ -15,8 +15,11 @@ class TestRegex(unittest.TestCase):
 
     def test_validate_valid(self):
         """Should pass validation for valid regular expression"""
-        self.assertTrue(re.validate("a*"))
-        self.assertTrue(re.validate("b|a?*"))
+        try:
+            re.validate("a*")
+            re.validate("b|a?*")
+        except Exception:
+            self.assertTrue(False)
```

Comment:
@eliotwrobson Would be better to use `self.fail()` with an appropriate error message here. See:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail

Reply:
@eliotwrobson On second thought, do we even need a `try..except` here? An exception raised during a test already counts as a failure.

Reply:
That is true, I added the fail statement so I suppose having the explicit error message is better. To be honest I thought about this but just defaulted to the more explicit check.

Reply:
@eliotwrobson Yes, but you're swallowing up the exception that was raised. From a debugging perspective, that would be annoying because you won't know what caused the failure. Was it a `TypeError`, `ValueError`, or something else?

Thinking it over, I would ask that the `try..catch` be removed. For me, practicality beats purity in this case, and I would much rather let any exceptions in my tests be raised naturally (so I have access to the full stack trace and all that good stuff).

Reply:
That makes sense, will go ahead and remove

Reply:
@caleb531 done here: https://github.com/caleb531/automata/pull/137/commits/d21a975d96ad4f0daf79e6f6f0b1e9e3c07abacb